src/assets/pages/blogposts/post-59.html

```diff
@@ -0,0 +1,25 @@
+<p><em>Este texto busca apresentar como, por meio da evolução da plataforma Querido Diário, cidadãos podem acessar, de forma mais ágil e acessível, dados públicos presentes em diários oficiais, promovendo a transparência e facilitando o acompanhamento das ações governamentais nos municípios brasileiros.​</em></p>
+<hr/>
+<p>Em busca por transparência em dados públicos, o Querido Diário torna mais compreensível e acessível o acesso a dados governamentais para os cidadãos. Dentro deste contexto, o grupo “Evolução do Querido Diário”, fruto da Residência em TIC promovida pela Brisa, em parceria com a Softex, o laboratório Lappis (UnB) e o Ministério da Ciência, Tecnologia e Inovação (MCTI), se destacou ao contribuir com novas funcionalidades para a evolução da plataforma.</p>
```

Comment:
É importante mencionar as pessoas que fazem parte do grupo citado. 
```suggestion
<p>Em busca por transparência em dados públicos, o Querido Diário torna mais compreensível e acessível o acesso a dados governamentais para os cidadãos. Dentro deste contexto, o grupo “Evolução do Querido Diário” (composto por Wildemberg Sales, Cristian Furtado, Pedro Cabeceira, Raíssa Oliveira, Ester Lino e Artur Ferreira), fruto da Residência em TIC promovida pela Brisa, em parceria com a Softex, o laboratório Lappis (UnB) e o Ministério da Ciência, Tecnologia e Inovação (MCTI), se destacou ao contribuir com novas funcionalidades para a evolução da plataforma.</p>
```